### PR TITLE
Oasis: Added description about the backup feature

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

File: 3.6/oasis/backups.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+description: Quick guide about backups in Oasis for ArangoDB.
+title: Backups in Oasis
+---
+
+# How to create backups
+
+To backup data in Oasis for an ArangoDB installation, navigate to the backup section of your deployment created previously.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:9]: Backups section

capitalise & bold "Backups"

File: 3.6/oasis/backups.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+![Backup ArangoDB](../images/oasis-backup-section.png)
+
+There are two types of backups. Periodic backups taken through backup policies and manual backups taken by hand.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:13]: There are two ways to create backups.
Create periodic backups using a <b>Backup policy</b>, or create a backup manually.

(if is not really a type)

File: 3.6/oasis/backups.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Periodic backups
+
+Periodic backups are taken on a given timeframe via a Backup Policy. To see when the new backup is due, observe the schedule section.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:17]: are created at a given schedule...

File: 3.6/oasis/backups.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+![Backup Policy schedule](../images/oasis-backup-policy-schedule.png)
+
+Upon creating a deployment, by default, a new policy is created which will take backups of a deployment hourly every two hours. To edit this policy (or any policy), highlight it in the row above and hit the pencil icon.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:21]: When a new deployment is created, a default backup policy is created for it as well.
This policy will create backups every two hours.....

File: 3.6/oasis/backups.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+It's also possible to create a backup on demand. To do this, click **Back up now**.
+
+![Back up Now](../images/oasis-back-up-now.png)
+
+![Back up Now Dialog](../images/oasis-back-up-now-dialog.png)
+
+## Uploading backups
+
+By default a backup is not uploaded to the cloud, meaning it's ephemeral. It will go away and not persist (by default in 6 hours). To keep a backup for a longer period of time, upload it to the cloud storage. Uploaded backups are required for [cloning](#how-to-clone-a-deployment-using-a-backup).
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:37]: By default a backup is not uploaded to the cloud, instead it remains on the servers of the deployment.
To make a backup that is resilient against server (disk) failures, upload the backup to cloud storage.

File: 3.6/oasis/backups.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+# Restoring from backup
+
+To restore a database from a backup, highlight the desired backup and click on the restore icon.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:41]: Add warning that:

1) All current data will be lost
2) During restore the deployment is temporarily not available

File: 3.6/oasis/backups.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+# How to clone a deployment using a backup
+
+1. Highlight the backup you wish to clone from and hit **Clone backup to new deployment**
+![Oasis Clone Deployment From Backup](../images/oasis-clone-deployment-from-backup.png)
+1. The view should navigate to the new deployment being bootstrapped
+![Oasis Cloned Deployment](../images/oasis-cloned-deployment.png)
+
+## Details about the cloned deployment
+
+This deployment will have the exact same features as the previous deployment
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:60]: The cloned deployment ....

File: 3.6/oasis/backups.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+including server specs, mode, provider, project and organization. The data
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:61]: including node size, model, cloud provider & region.

File: 3.6/oasis/backups.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+contained in the backup will be restored to this new deployment.
+
+The *password* for this deployment, however, will be generated anew.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:64]: The root password ... will be different.

File: 3.6/oasis/backups.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+This feature is also available through Oasisctl, including an optional parameter to specify a new region. The provider stays the same.
```
Comment (ewoutp) [inline 3.6/oasis/backups.md:66]: Remove "including an optional parameter to specify a new region. The provider stays the same."

File: 3.6/oasis/backups.md (hunk 4 above)
Comment (Skarlso) [reply 3.6/oasis/backups.md:21]: I thought that might be weird phrasing. It was taken from this:
```
Hourly every 2 hour(s)
```

Which is displayed in the schedule.

**Pull request merged**